scout/server/blueprints/variant/templates/variant/sv-variant.html

```diff
@@ -264,9 +264,12 @@
 	    {% endif %}
           </td>
 	  <td>
-	    <a class="btn btn-outline-secondary form-control" href="https://decipher.sanger.ac.uk/browser#q/{{ variant.chromosome }}:{{ variant.position }}-{{ variant.end }}%20/location/{{ variant.chromosome }}:{{ variant.position }}-{{ variant.end }}" target ="_blank">DECIPHER {% if case.genome_build == 38 %} (GRCh37){% endif %}</a>
-
-	  </td>
+	    {% if case.genome_build == 38 %}
```

Comment:
This should actually also use links.py, but it was already like this. Feel free to unify, but not required!